src/main/java/DeleteContactPage.java

```diff
@@ -0,0 +1,30 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class DeleteContactPage extends BasePage{
+    public DeleteContactPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(id = "check-box-remove-contact")
+    private WebElement checkBox;
+
+    @FindBy(id = "submit-remove")
+    private WebElement buttonDeleteElem;
```

Comment:
I don't think it was worth cutting a few characters

Reply:
I fix it